src/novel_applications/payment/private_payment.md

```diff
@@ -0,0 +1,167 @@
+# Private Payment
+
+![Private payment header image](../../assets/images/private_payment.png)
+
+## Blockchains as Payment Infrastructure
+
+Efficient payments are the economic backbone of human societies. They enable the exchange of value for goods and services, facilitate social functioning (e.g., participating in activities like dining out), and support financial interactions.
+
+Since the inception of Bitcoin, the first blockchain, payments have been the flagship use case. With further developments like Ethereum, alternative L1s (Polygon, Tron, Celo, BNB), and innovations like rollups (Base, Starknet, Scroll), payments have become faster and cheaper.
+
+The lowered transaction fees offered by these alternative chains make them more suitable for routine, person-to-person payments and small-scale commercial transactions. This "real-world" adoption is evident from [data on stablecoin usage across these chains](https://app.artemisanalytics.com/stablecoins?chain=all&stablecoinMetric=STABLECOIN_MC). The term "real world" in this context refers primarily to user-to-user transfers, as opposed to institutional transactions. This interpretation is supported by the notably lower average transaction sizes compared to Ethereum (~$15 vs ~$100k).
+
+While blockchains have largely solved the problems of high gas fees and slow execution/finality through the introduction of alternative layer-1 and layer-2 solutions, two critical issues continue to prevent mass adoption:
+
+1. **User Experience (UI/UX)**: Despite technological advancements, many blockchain applications still suffer from poor UI/UX design. The complexity of interacting with blockchain systems remains a significant barrier for non-technical users, often involving confusing wallet interfaces, complicated address systems, sensitive seed phrases and unintuitive transaction processes.
```

Comment:
I would name as 1. still scalability. Obviously, Bitcoin and Ethereum don't scale enough to provide fast and cheap payments. But even Tron or Celo with ~100 TPS do not nearly scale enough to provide a payment system for the masses. 

Miden on the other hand, having parallel transaction execution and 1000+ TPS will be able to compete with VISA and PayPal. 

We can list the UI/UX point as third point.